src/gz_TEST.cc

```diff
@@ -129,6 +131,29 @@ TEST(CmdLine, GazeboServer)
   }
 }
 
+/////////////////////////////////////////////////
+TEST(CmdLine, SimtimeArgument)
+{
+  std::string cmd = kGzCommand + " -r -v 4 --iterations 100 --initial-sim-time 1000.0 " +
```

Comment:
I would try with a non-zero decimal, like `1000.5` to confirm that the fractional seconds are handled properly

Reply:
Done https://github.com/gazebosim/gz-sim/pull/1801/commits/6c2a1344dc2db641a64f7d68a8816ae74a3f6567